consensus/spos/consensusCore.go

```diff
@@ -239,6 +239,131 @@ func (cc *ConsensusCore) EquivalentProofsPool() consensus.EquivalentProofsPool {
 	return cc.equivalentProofsPool
 }
 
+// SetBlockchain sets blockchain handler
```

Comment:
are these only used for tests? perhaps move them to export_test.go

Reply:
they are also used in testscommon, so it doesn't work if we move them to export_test.og